packages/next/taskfile-swc.js

```diff
@@ -72,7 +72,7 @@ module.exports = function (task) {
           targets: {
             // Same version defined in packages/next/package.json#engines
             // Currently a few minors behind due to babel class transpiling
-            node: '16.14.0',
+            node: '16.8.0',
```

Comment:
can we also bump this node version?

Reply:
@huozhi This seems to break the tests as seen in [d6bd448](https://github.com/vercel/next.js/pull/54331/commits/d6bd4485fdb425969938e2ac4bbae9f19c1ae003) <picture data-single-emoji=":thonk:" title=":thonk:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/thonk/d620c05f0e664ad6.png" alt=":thonk:" align="absmiddle"></picture>

Reply:
We also have comment `// Same version defined in packages/next/package.json#engines` above, maybe add a TODO comment to let us follow up later?

Reply:
@huozhi I added a comment in [ee1daca](https://github.com/vercel/next.js/pull/54331/commits/ee1daca8272ba8dc1d50370be6d8bdb7cc5ce9ae)